docs/docs/overview/quickstart/get-started-with-python.md

```diff
@@ -0,0 +1,198 @@
+---
+sidebar_position: 10
+sidebar_label: Get started with Python
+---
+
+# Python Quickstart
+
+Welcome to the Python Quickstart guide for Novu, a powerful notification service that enables you to send multi-channel (SMS, Email, Chat, Push) notifications from your Python applications. In this Quickstart, you'll learn how to seamlessly integrate Novu into your Python project and perform various essential tasks. Let's get started!
+
+## Prerequisites
+
+Before diving into the Quickstart, make sure you have the following:
+
+- Python3 installed on your development machine.
+- A Novu account. If you don't have one, sign up for free at [web.novu.co](https://web.novu.co)
+
+You can also [view the completed code](https://web.novu.co) of this quick start in a GitHub repo.
+
+### Install and Set Up Novu in your Python Project
+
+First, you must install the Novu package in your Python project. You can install the Novu package in your project in two ways. Open your terminal and run the following command:
+
+Via pip
+
+```bash
+pip install novu
+```
+
+Via poetry
+
+```bash
+poetry add novu
+```
+
+Once installed, you can import Novu into your project and initialize it using your Novu account credentials. This step establishes a connection between your app and the Novu notification service.
+
+```python
+from novu.config import NovuConfig
+
+NovuConfig().configure("https://api.novu.co", "<YOUR_NOVU_API_KEY>")
+```
+
+Replace the `<YOUR_NOVU_API_KEY>` value with the authentic key from the **API Key** section of your [Novu Dashboard](https://web.novu.co/settings).
```

Comment:
Replace the `<YOUR_NOVU_API_KEY>` value with your own API key that you got from the **API Key** section of your [Novu Dashboard](https://web.novu.co/settings).